src/test/java/com/google/sps/CapstoneAuthTest.java

```diff
@@ -0,0 +1,103 @@
+package com.google.sps;
+
+import java.io.IOException;
+
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import static org.junit.Assert.*;
+
+import static org.mockito.Mockito.*;
+
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import com.google.cloud.datastore.Key;
+
+import com.google.sps.CapstoneAuth;
+import static com.google.sps.ChatWebSocket.*;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.firebase.auth.UserRecord;
+import com.google.firebase.auth.FirebaseAuthException;
+
+
+@PrepareForTest({CapstoneAuth.class, UserRecord.class, FirebaseAuth.class,
+    FirebaseToken.class})
+@RunWith(PowerMockRunner.class)
+public class CapstoneAuthTest {
+
+    public static final String TEST_ID_TOKEN = "1234testtoken";
+    public static final String TEST_UID = "testuid";
+    public static final String TEST_FALSE_UID = "testfalseuid";
+    public static final String TEST_EMAIL = "test@email.com";
+
+    private CapstoneAuth obj;
```

Comment:
nit: Instead of naming it obj, name is capstoneAuth.

Reply:
That makes sense, thanks for catching that!